index.html

```diff
@@ -2318,68 +2256,147 @@ <h2>
 
-        <li>
+      <p>
 Method-specific endpoint authentication MUST be discussed. Where
 <a>DID methods</a> make use of <a>DLTs</a> with varying network topology, sometimes
 offered as "light node" or "<a href="https://en.bitcoin.it/wiki/Thin_Client_Security">thin client</a>"
 implementations to reduce required computing resources, the security
 assumptions of the topology available to implementations of the <a>DID
 method</a> MUST be discussed.
-        </li>
+      </p>
 
-        <li>
+      <p>
 <a>DID methods</a> MUST discuss the policy mechanism by which <a>DIDs</a> are
 proven to be uniquely assigned. A <a>DID</a> fits the functional definition
 of a URN as defined in [[RFC8141]] &mdash; a persistent identifier that is
 assigned once to a resource and never reassigned. In a security
 context this is particularly important since a <a>DID</a> may be used to
 identify a specific party subject to a specific set of authorization
 rights.
-        </li>
+      </p>
 
-        <li>
+      <p>
 <a>DID methods</a> that introduce new authentication <a>service endpoint</a>
-types (Section <a href="#service-endpoints"></a>) SHOULD consider the
+types (see <a href="#service-endpoints"></a>) SHOULD consider the
 security requirements of the supported authentication protocol.
-        </li>
-      </ol>
+      </p>
+
     </section>
 
+    <section class='normative'>
+      <h2>
+Privacy Requirements
+      </h2>
+      <p>
+<a>DID method</a> specifications MUST include their own Privacy
+Considerations sections, if only to point to
+<a href="#privacy-considerations"></a>.
+      </p>
+
+      <p>
+The <a>DID method</a> specification's Privacy Considerations section
+MUST discuss any subsection of
+section 5 of [[RFC6973]] that could apply in a method-specific
+manner. The subsections to consider are: surveillance, stored data
+compromise, unsolicited traffic, misattribution, correlation,
+identification, secondary use, disclosure, exclusion.    
+      </p>
+    </section>
+
+  </section>
+
+  <section class='normative'>
+    <h1>
+DID Resolvers
+    </h1>
+
+    <p>
+A <a>DID resolver</a> is a software or hardware component with an API for resolving
+<a>DIDs</a> of at least one <a>DID method</a>. It
+executes the corresponding <a>DID method</a>'s <a href="#read-verify">Read</a> operation to
+obtain the authoritative <a>DID document</a>.
+    </p>
+
+    <p>
+The interfaces and algorithms for resolving <a>DIDs</a> and dereferencing <a>DID URLs</a> are
+specified in [[DID-RESOLUTION]].
+    </p>
+
+  </section>
+
+  <section class="informative">
+    <h1>
+Security Considerations
+    </h1>
+
+    <p>
+NOTE TO IMPLEMENTERS: During the Implementer’s Draft stage, this
+section focuses on security topics that should be important in early
+implementations. The editors are also seeking feedback on threats and
+threat mitigations that should be reflected in this section or
+elsewhere in the spec. As the root identifier records for <a>decentralized
```

Comment:
This isn't quite a sentence. I can't tell what it's meant to say, so I can't provide a suggested revision --
```
As the root identifier records for <a>decentralized
identifiers</a> (DIDs) and <a>DID documents</a> are a vital component of
<a>decentralized identity management</a>.
```

Reply:
I agree. I suspect the whole paragraph is old and might not be relevant any more, but rewriting or removing it probably needs to be a different PR

Reply:
opened #115 to track